packages/esm-patient-forms-app/src/index.ts

```diff
@@ -55,7 +55,7 @@ function setupOpenMRS() {
         name: 'forms-widget',
         slot: 'patient-chart-summary-dashboard-slot',
         order: 5,
-        load: getAsyncLifecycle(() => import('./forms/forms-overview.component'), options),
```

Comment:
Is `forms-overview.component` unused? Can we delete it?

Reply:
Its used renamed it to `form-summary-dashboard.component.tsx`